File: backend/search/brand.go
package search

// Brand defines Brand type
type Brand struct {
	ID       int64  `json:"id" mapstructure:"id"`
	Version  int64  `json:"version" mapstructure:"version"`
	Slug     string `json:"slug" mapstructure:"slug"`
	Name     string `json:"name" mapstructure:"name"`
	ImageURL string `json:"image_url" mapstructure:"image_url"`
}

File: frontend/src/components/brand.js
import React, { useState } from 'react';
import Row from 'react-bootstrap/Row'
import Form from 'react-bootstrap/Form'
import Card from 'react-bootstrap/Card'
import CardGroup from 'react-bootstrap/CardGroup'

function Brand(props) {
    let cards = props.data.map(el => (
        <CardGroup>
            <Card key={el.id}>
                <Card.Img variant="top" height={200} width={300} src={el.image_url} />
                <Card.Body>
                    <Card.Title>{el.name}</Card.Title>
                </Card.Body>
                <Card.Footer>
                    <small className="text-muted">{el.slug}</small>
                </Card.Footer>
            </Card>
        </CardGroup>
    ))
    return (
        <Row>
            {cards}
        </Row>
    );
}

export default Brand